Tools/TestWebKitAPI/Tests/WebKitCocoa/WKWebViewConfiguration.mm

```diff
@@ -172,3 +172,25 @@ HTTPServer server([&] (Connection connection) {
     NSString *bodyHTML = [webView stringByEvaluatingJavaScript:@"document.body.innerHTML"];
     EXPECT_WK_STREQ(bodyHTML, @"<table></table>");
 }
+
+TEST(WebKit, ConfigurationDisableJavaScriptSVGAnimateElement)
```

Comment:
Why is not this a layout test?

Reply:
the pref needs to be enabled in WkWebViewconfiguration